share/pixmaps/dash-hicolor-scalable.svg

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Should this specifically be licensed under MIT?

Reply:
I don't know the answer to that.

Reply:
@PastaPastaPasta -- Is there a request here for me? You want me to add some metadata or some such? Or just stick a license file in the same directory?

Reply:
Hmm... I'm not sure if we need this since it's kind of set globally here https://github.com/dashpay/dash#license and https://github.com/dashpay/dash/blob/master/COPYING. The only 2 svg files with an explicit licence I found are https://github.com/dashpay/dash/blob/master/contrib/macdeploy/background.svg?short_path=9c330af#L5-L9 and https://github.com/dashpay/dash/blob/master/src/qt/res/src/bitcoin.svg?short_path=14cf0c5#L4 (I wonder why we have it...) but it's also MIT there and these files are rather old. Other/newer files in https://github.com/dashpay/dash/tree/master/src/qt/res/src have no explicit info about their licence for example. Thoughts @nmarley ? (and maybe @strophy)

Reply:
I didn't think there was a need to explicitly add a license for these since they're a small part of a larger work which is already MIT licensed unless otherwise stated.

Is there some reason (or just precedent) why you raised this concern @pastapastapasta ?

Reply:
Not really, I just know that all of our source files specifically state MIT so I just wanted to bring up the thought